bin/tape

```diff
@@ -41,18 +45,36 @@ if (typeof opts.ignore === 'string') {
     var matcher = ignore.createMatcher(ignoreStr);
 }
 
-opts._.forEach(function (arg) {
+var files = opts._.reduce(function (result, arg) {
     // If glob does not match, `files` will be an empty array.
     // Note: `glob.sync` may throw an error and crash the node process.
     var files = glob.sync(arg);
 
-    if (!Array.isArray(files)) {
-        throw new TypeError('unknown error: glob.sync did not return an array or throw. Please report this.');
+    if (files[0] === 'test/import/mjs-e.mjs') {
```

Comment:
i assume this was just for testing

Reply:
argh, at least three times i looked at that and thought "better not commit" that. my bad